lib/rules/selector-max-class/README.md

```diff
@@ -0,0 +1,51 @@
+# selector-max-class
+
+Limit the number of classes in a [complex selector](https://www.w3.org/TR/selectors4/#complex).
+
+```css
+   div .foo.bar[data-val] > a.baz {}
```

Comment:
Let's shift this so the `<div>` is flush to the left. We only indent when there's not space for the arrow.

There's a tonne of documentation in stylelint and so we try to keep it ship-shape :)